src/plone/base/interfaces/controlpanel.py

```diff
@@ -1,7 +1,10 @@
+from xmlrpc.client import boolean
```

Comment:
```suggestion
```
I think this boolean thing was imported by the editor but should not actually be here.